### PR TITLE
Fix #59, recognizing queries without quotes

### LINKED ISSUE #59: Borg should recognize queries without quotes
ie.

```borg "git delete branch"```

Should be identical to
```borg git delete branch```

### PR DESCRIPTION
**Pull request by code-inflation**

### CODE REVIEW

**Comment by code-inflation** [comment]
breaks commands like "borg login asdf"

**Comment by crufter** [comment]
@code-inflation yes, the PR is almost correct, but the `searchArgs := strings.Join(flag.Args(), " ")` should be inside the `Query` branch

File: main.go [hunk 1]
```diff
@@ -24,8 +26,9 @@ func main() {
 	}
 
 	var err error
-	if c, ok := commands.Commands[flag.Arg(0)]; !ok {
-		err = commands.Query(flag.Arg(0))
```
Comment (crufter) [inline main.go]: This should be the only line modified I think: ```err = commands.Query(strings.Join(flag.Args(), " "))```

**Comment by code-inflation** [comment]
Thank you, I will try that

**Comment by crufter** [comment]
Looks good to me

**Pull request merged**